buildenv/jenkins/README.md

```diff
@@ -92,6 +92,14 @@ You can also request a Pull Request build from the extensions repos or openj9-om
 - To trigger a Copyright Check
    - `Jenkins copyright check`
 
+- To trigger a SignedOffBy Check
+   - `Jenkins signed off by check`
+   - add a signedOffByCheck file
+   - check the index of the last blank line
+   - everything after the last blank line are regarded as footer
+   - as long as one of the lines in the footer contains right format of signoff
```

Comment:
how about: "one of the lines in the footer must contain a correctly formatted signoff"